Src/Support/Google.Apis.Auth.Tests/OAuth2/TokenFakes.cs

```diff
@@ -146,8 +146,15 @@ protected override async Task<HttpResponseMessage> SendAsyncCore(HttpRequestMess
     /// </summary>
     internal class FetchesTokenMessageHandler : CountableMessageHandler
     {
-        protected override Task<HttpResponseMessage> SendAsyncCore(HttpRequestMessage request, CancellationToken taskCancellationToken) =>
-            Task.FromResult(new HttpResponseMessage()
+        public string LatestRequestContent { get; private set; }
+
+        protected override async Task<HttpResponseMessage> SendAsyncCore(HttpRequestMessage request, CancellationToken taskCancellationToken)
+        {
+            if (request.Content != null)
```

Comment:
Perhaps we should set LatestRequestContent either way, setting it to null if there was no content?

Reply:
Yep, done.